setup.sh

```diff
@@ -123,11 +136,17 @@ apply () {
         if [ $DEV_ENVIRONMENT = 'true' -a "$1" = 'infra-networking' ] ; then
                 import_shared_dev_route53
         fi
+
+        # store output in a subprocess
```

Comment:
I think this comment is no longer needed?

Reply:
👍